zadarapy/session.py

```diff
@@ -153,9 +157,23 @@ def __init__(self, host=None, port=None, key=None, configfile=None,
                         false_values:
                     self.zadara_secure = False
 
+        if verify_ssl is False:
+            self.zadara_verify_ssl = False
+        elif os.getenv('ZADARA_VERIFY_SSL') is not None:
+            if os.getenv('ZADARA_VERIFY_SSL').lower() in false_values:
+                self.zadara_verify_ssl = False
+        elif self._config is not None:
+            if self._config.defaults().get('verify_ssl', None) is not None:
```

Comment:
to avoid double if
```python
if self._config.defaults().get('verify_ssl', 'true').lower() in false_values:
```
or better (also applies to `os.getenv`):
```python
self.zadara_verify_ssl = self._config.defaults().get('verify_ssl', 'true').lower() not in false_values
```